web/cobrands/fixmystreet.com/base.scss

```diff
@@ -49,6 +50,66 @@ svg|g.site-logo__svg {
     @include svg-background-image("/cobrands/fixmystreet/images/site-logo");
 }
 
+body.frontpage {
+  $fms-logo-width: 150px;
+  $fms-logo-height: 35px;
+  $mysociety-badge-width: 90px;
```

Comment:
Maybe it’s just me, but I find this a smidge too large, it feels like it’s overpowering the FMS logo beside it:

![Screen Shot 2023-01-31 at 16 09 10](https://user-images.githubusercontent.com/739624/215815871-83c9bb96-5bee-4fdf-87c4-b842f30ef52f.png)

Here’s how it looks reduced from 90px to 75px – a bit more balanced?

![Screen Shot 2023-01-31 at 16 09 04](https://user-images.githubusercontent.com/739624/215815949-bb1e6d51-ecb4-45bb-a37c-1de9808c0b2d.png)